keras_core/layers/convolutional/conv_transpose_test.py

```diff
@@ -361,7 +361,7 @@ def test_conv2d_transpose(
             dilation_rate=dilation_rate,
         )
 
-        inputs = np.random.normal(size=[2, 8, 8, 4])
+        inputs = np.random.normal(size=[2, 14, 14, 4])
```

Comment:
Can you add a case for stride > input size? Also prefer adding new cases rather than changing existing parameters, it increases coverage.